vpr/src/place/timing_place_lookup.cpp

```diff
@@ -605,6 +590,17 @@ static void fix_uninitialized_coordinates(vtr::Matrix<float>& delta_delays) {
     }
 }
 
+static void fill_impossible_coordinates(vtr::Matrix<float>& delta_delays) {
+    //Set any empty delta's to the average of it's neighbours
```

Comment:
Update comment (refers to empty deltas, not impossible ones).
Say why 5 is chosen as max distance here.